### DISCUSSION TITLE
Ch 15 minor error in code

### DISCUSSION THREAD

**Discussion by ghost** [r-lidar/lidR#457] [General]
Hello,

I have been working through all the examples in this book (very happy with this package). 

Just wanted to point out a minor error. There is an odd number of " so the code won't run as is for the first example of the chapter

```r
newname <-  tools::file_path_sans_ext(basename(file))
newname <- paste0("folder/, newname, "_colored.laz")
las <- readLAS(file)
las <- merge_spatial(las, rgb)
writeLAS(las, newname)
```

I just changed the output directory information and it worked fine.

I also am confused on 15.3 for the user defined function

```r
dtm_norm_color_chunk <- function(chunk, rgbmap) # user defined function
```

I do not see anywhere where 'chunk' is defined. I want to assume that it could be something like: 

```r
chunk <- readLAS(here("file/path"))
```

and that it is the first file of the desired catalog, but this is unclear. 

thanks for the hard work! 
cheers, Mackenzie

**Comment by ghost** [comment]
I figured out the chunk part so no need to address that anymore!

**Reply by Jean-Romain** [reply]
great !

**Comment by Jean-Romain** [comment]
> There is an odd number of " 

Fixed thank you

> I do not see anywhere where 'chunk' is defined. 

`chunk` is the first input of the user-defined function (here `dtm_norm_color_chunk`)
